src/index.html

```diff
@@ -20,8 +24,354 @@
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rate-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-money">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rate-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-money">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rate-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-money">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The product cards are repeated with the same content. If this is for demonstration purposes, consider varying the content to better simulate a real catalog. Otherwise, ensure that each card represents a unique product.